docs/hooks/index.md

```diff
@@ -35,64 +27,48 @@ An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types
 1. Notifying downstream consumers:
    Running a post-merge hook to trigger an Airflow DAG or to send a Webhook to an API, notifying it of the change that happened
 
-For step-by-step examples of hooks in action check out the [lakeFS samples repository](https://github.com/treeverse/lakeFS-samples/).
+For step-by-step examples of hooks in action check out the [lakeFS Quickstart](/quickstart/actions-and-hooks.html) and the [lakeFS samples repository](https://github.com/treeverse/lakeFS-samples/).
 
-## Supported Events
+## Overview
 
-| Event                | Description                                                                    |
-|----------------------|--------------------------------------------------------------------------------|
-| `pre-commit`         | Runs when the commit occurs, before the commit is finalized                    |
-| `post-commit`        | Runs after the commit is finalized                                             |
-| `pre-merge`          | Runs on the source branch when the merge occurs, before the merge is finalized |
-| `post-merge`         | Runs on the merge result, after the merge is finalized                         |
-| `pre-create-branch`  | Runs on the source branch prior to creating a new branch                       |
-| `post-create-branch` | Runs on the new branch after the branch was created                            |
-| `pre-delete-branch`  | Runs prior to deleting a branch                                                |
-| `post-delete-branch` | Runs after the branch was deleted                                              |
-| `pre-create-tag`     | Runs prior to creating a new tag                                               |
-| `post-create-tag`    | Runs after the tag was created                                                 |
-| `pre-delete-tag`     | Runs prior to deleting a tag                                                   |
-| `post-delete-tag`    | Runs after the tag was deleted                                                 |
+An _Action_ defines one or more _Hooks_ to execute. lakeFS supports three types of hook: 
```

Comment:
Not a fan of caps to define things - that's why we capitalize.  Calling it an "Action" rather than an "action" here creates some expectation to call it an "Action" throughout documentation, which is a painful chore.

Reply:
ack. changed.